scripts/local.py

```diff
@@ -19,26 +20,32 @@
 bs_local_args = { "key": BROWSERSTACK_ACCESS_KEY }
 
 # Starts the Local instance with the required arguments
+try:
+  bs_local.start(**bs_local_args)
+except Exception:
+  bs_local.stop()
+
 bs_local.start(**bs_local_args)
 
 # Check if BrowserStack local instance is running
 print(bs_local.isRunning())
 
 desired_cap = {
-  'browserName': 'iPhone',
-  'device': 'iPhone 11',
-  'realMobile': 'true',
-  'os_version': '14.0',
-  'name': 'BStack-[Python] Sample Test', # test name
-  'build': 'BStack Build Number 1', # CI/CD job or build name
-  'browserstack.local': 'true',
-  'browserstack.user': BROWSERSTACK_USERNAME,
-  'browserstack.key': BROWSERSTACK_ACCESS_KEY
+    "os" : "OS X",
+    "osVersion" : "Sierra",
+    "buildName" : "Final-Snippet-Test",
```

Comment:
Build Name and Session Name should remain as it is

Reply:
It can be different